app/src/main/java/com/vero/tasky/auth/domain/usecase/LoginUseCases.kt

```diff
@@ -1,6 +1,7 @@
 package com.vero.tasky.auth.domain.usecase
 
 import com.vero.tasky.auth.domain.usecase.password.ValidatePasswordUseCase
+import com.vero.tasky.core.domain.usecase.ValidateEmailUseCase
 
```

Comment:
Why do you have the Password one inside a password package?

Reply:
Since the password usecase also has one more class to use PasswordValidationResult, I thought it will be better to save these classes in a separate package 
![image](https://github.com/AndreVero/Tasky/assets/39765531/39617d6b-aabf-4cf8-8e35-d727ffc21d74)

Reply:
Ah gotcha
Makes sense